herbert_core/utilities/classes/@object_lookup/object_repmat.m

```diff
@@ -0,0 +1,177 @@
+function obj_out = object_repmat (obj, varargin)
+% Return a given object array from the original set of arrays
+%
+% Apply to a specific array or set of arrays in in the object_lookup:
+%   >> obj_out = object_repmat (obj, iarray, '-repeat', sz_repmat)
+%
+% Apply to the whole object_lookup:
+%   >> obj_out = object_repmat (obj, '-repeat', sz_repmat)
```

Comment:
`-arg` is reserved for `true`/`false` arguments. 
https://github.com/pace-neutrons/Horace/blob/master/documentation/adr/0022-use-keyword-args.md

Reply:
Comment out of date given a change made to the syntax of object_lookup.